### PR TITLE
Catch exceptions by ref on windows processes

### PR DESCRIPTION
**Pull request by fmanco**

### CODE REVIEW

File: osquery/core/windows/process.cpp [hunk 1]
```diff
@@ -137,9 +137,9 @@ std::shared_ptr<PlatformProcess> PlatformProcess::getLauncherProcess() {
   try {
     handle = reinterpret_cast<HANDLE>(static_cast<std::uintptr_t>(
         std::stoull(*launcher_handle, nullptr, 16)));
-  } catch (std::invalid_argument e) {
+  } catch (std::invalid_argument& e) {
```
Comment (akindyakov) [inline osquery/core/windows/process.cpp]: It would be a bit more safe and correct to catch it by the `const` ref.

**Comment by akindyakov** [comment]
Looks good 👍

**Comment by fmanco** [comment]
jenkins test windows

**Comment by fmanco** [comment]
jenkins test windows

**Pull request merged**